cooperative_lanechange/src/cooperative_lanechange.cpp

```diff
@@ -225,25 +246,37 @@ namespace cooperative_lanechange
             }
         }
         if(foundRoadwayObject){
+            ROS_DEBUG_STREAM("Found Roadway object");
             //get current_gap
+            ROS_DEBUG_STREAM("veh2_lanelet_id:" << veh2_lanelet_id << "veh2_downtrack" << veh2_downtrack);
+            
             double current_gap = find_current_gap(veh2_lanelet_id,veh2_downtrack);
+            ROS_DEBUG_STREAM("Current gap:"<<current_gap);
             //get desired gap - desired time gap (default 3s)* relative velocity
             double relative_velocity = current_speed_ - veh2_speed;
+            ROS_DEBUG_STREAM("Relative velocity:"<<relative_velocity);
             double desired_gap = desired_time_gap_ * relative_velocity;      
-            
-            if(current_gap > desired_gap){
-                negotiate = false;  //No need for negotiation
+            ROS_DEBUG_STREAM("Desired gap:"<<desired_gap);
+            if(desired_gap< 5.0){
+                desired_gap = 5.0;
             }
+            // if(current_gap > desired_gap){
```

Comment:
If we're going to leave this as-is let's write up a user story to patch both this and the above desired gap calculation.

Reply:
Yes that makes sense. An issue we were seeing while calculating the gap was that sometimes the map has been modified in a way that there is no path from veh2 to veh1. 
Currently we have had to modify the map in a way that lanechange is possible in both direction, so that a path exists.
Would it make sense to use the (x,y) map coordinates to calculate the gap? It should always be larger than the downtrack distance, i think

Reply:
Hmm, not really sure about using cartesian coordinates, there's some tradeoffs there too. I think ideally we'd fix what the underlying issue is with the lanelet graph, this kind of a gap should be something we're able to compute. Is the issue particular to our map at summit or is it more related to how we use lanelet2 in general?

Reply:
That makes sense, the issue was particular to the map at summit and even tfhrc